### PR TITLE
Refactor storage service

### PR DESCRIPTION
**Pull request by basultobdKs**
1. Pass the storage service to as paramater to the other services.
2. Delete `Dashboard` and `AppointmentCard` components.
3. Change route name for render `Calendar` page component, from /`dashboard` to /`calendar`.
4. Now we use the _Services_ directly instead _Providers_ inside the components. Due this I remove all the `Consumers` and `Providers` components for `Booking`, `Profile`, `Room` and `User` services.
5. Remove prop drilling from `Calendar` page component. Finally!

 As per in #76

### CODE REVIEW

File: src/services/User/UserService.js [hunk 1]
```diff
@@ -92,7 +97,7 @@ export const UserService = () => {
    * @param {string} authToken - authorization token.
    */
   // TODO: @returns {NotContentResponse} - request response.
-  const deleteOne = async (id, authToken) => {
+  const deleteOne = async id => {
```
Comment (mishelashala) [inline src/services/User/UserService.js]: rename to `deleteOneById`

File: src/services/User/UserService.js [hunk 2]
```diff
@@ -59,7 +64,7 @@ export const UserService = () => {
    * @param {number} id - user id.
    * @returns {User}
    */
-  const updateOne = async (user, id, authToken) => {
+  const updateOne = async (user, id) => {
```
Comment (mishelashala) [inline src/services/User/UserService.js]: rename to `updateOneById`

File: src/services/User/UserService.js [hunk 3]
```diff
@@ -33,10 +39,9 @@ export const UserService = () => {
    * Return the found user information using the id.
    * @memberof UserSservice
    * @param {number} id - user id.
-   * @param {string} authToken - authorization token.
    * @returns {User} - found user information.
    */
-  const getOne = async (id, authToken) => {
+  const getOne = async id => {
```
Comment (mishelashala) [inline src/services/User/UserService.js]: rename to `getOneById`

File: src/services/Room/RoomService.js [hunk 4]
```diff
@@ -135,10 +137,9 @@ export const RoomService = () => {
    * Delete a room by id.
    * @memberof RoomService
    * @param {number} id - room id.
-   * @param {string} authToken - authorization token.
    */
   //  TODO: @returns {NotContentResponse} - request response.
-  const deleteOne = async (id, authToken) => {
+  const deleteOne = async id => {
```
Comment (mishelashala) [inline src/services/Room/RoomService.js]: rename to `deleteOneById`

File: src/services/Room/RoomService.js [hunk 5]
```diff
@@ -107,10 +110,9 @@ export const RoomService = () => {
    * @memberof RoomService
    * @param {Room} room - room object.
    * @param {number} id - room id.
-   * @param {string} authToken - authorization token.
    * @returns {RoomResponse} - room updated information.
    */
-  const updateOne = async ({ name, color }, id, authToken) => {
+  const updateOne = async (id, { name, color }) => {
```
Comment (mishelashala) [inline src/services/Room/RoomService.js]: rename by `updateOneById`

File: src/services/Room/RoomService.js [hunk 6]
```diff
@@ -60,10 +65,9 @@ export const RoomService = () => {
    * Return the found Room information using the id.
    * @memberof RoomService
    * @param {number} id - room id.
-   * @param {string} authToken - authorization token.
    * @returns {RoomResponse} - found room information.
    */
-  const getOne = async (id, authToken) => {
+  const getOne = async id => {
```
Comment (mishelashala) [inline src/services/Room/RoomService.js]: rename to `getOneById`

File: src/services/Profile/ProfileService.js [hunk 7]
```diff
@@ -73,13 +76,11 @@ export const ProfileService = () => {
    * Update the Profile information and return it.
    * @param {Profile} profile - profile information.
    * @param {number} id - profile id.
-   * @param {string} authToken - authorization token.
    * @returns {ProfileResponse} - profile updated information.
    */
-  const updateOne = async ({ time_zone, locale }, id, authToken) => {
+  const updateOne = async (id, { time_zone, locale }) => {
```
Comment (mishelashala) [inline src/services/Profile/ProfileService.js]: rename to `getOneById`

File: src/services/Profile/ProfileService.js [hunk 8]
```diff
@@ -27,10 +32,9 @@ export const ProfileService = () => {
    * Return the found profile information.
    * @memberof ProfileService
    * @param {number} id - profile id.
-   * @param {string} authToken - authorization token.
    * @returns {ProfileResponse} - found profile information.
    */
-  const getOne = async (id, authToken) => {
+  const getOne = async id => {
```
Comment (mishelashala) [inline src/services/Profile/ProfileService.js]: rename to `getOneById`

File: src/services/Booking/BookingService.js [hunk 9]
```diff
@@ -109,10 +111,9 @@ export const BookingService = () => {
    * Return the found booking information using the id.
    * @memberof BookingService
    * @param {number} id - booking id.
-   * @param {string} authToken - authorization token.
    * @return {BookingResponse} - found booking information.
    */
-  const getOne = async (id, authToken) => {
+  const getOne = async id => {
```
Comment (mishelashala) [inline src/services/Booking/BookingService.js]: rename to `getOneById`

File: src/services/Booking/BookingService.js [hunk 10]
```diff
@@ -211,10 +211,9 @@ export const BookingService = () => {
    * Delete a booking by id.
    * @memberof BookingService
    * @param {number} id - booking id.
-   * @param {string} authToken - authorization token.
    * @returns {NotContentResponse} - request response.
    */
-  const deleteOne = async (id, authToken) => {
+  const deleteOne = async id => {
```
Comment (mishelashala) [inline src/services/Booking/BookingService.js]: rename to `deleteOneById`

File: src/components/BookingsSideBar/BookingList/BookingList.jsx [hunk 11]
```diff
@@ -29,8 +29,8 @@ class BookingListComponent extends React.Component {
   getBookings = async () => {
     const stateBookings = this.state.bookingItems;
     let bookingItems = stateBookings.map(async book => {
-      const user = this.props.userService.getUser(book.user_id);
-      const room = this.props.roomService.getRoom(book.room_id);
+      const user = userService.getOne(book.user_id);
```
Comment (mishelashala) [inline src/components/BookingsSideBar/BookingList/BookingList.jsx]: all properties' models must be in `camelCase`

File: src/components/BookingsSideBar/BookingList/BookingItem/BookingItem.jsx [hunk 12]
```diff
@@ -52,11 +52,15 @@ class BookingItemComponent extends React.Component {
 
   handleOnDelete = async () => {
     const { booking } = this.props;
+    console.log(booking);
```
Comment (mishelashala) [inline src/components/BookingsSideBar/BookingList/BookingItem/BookingItem.jsx]: delete logs

Comment (basultobdKs) [reply src/components/BookingsSideBar/BookingList/BookingItem/BookingItem.jsx]: Fixed in 7a3d718

**Pull request merged**